src/model/Entity/EntityFactory.php

```diff
@@ -1,13 +1,18 @@
 <?php
-namespace base\model;
+namespace base\model\entity;
 
 class EntityFactory
 {
 	protected $directory = "model";
 	
 	public function getEntity( $name )
 	{
-		$class = "project\\".$this->directory."\\".$name;
+		if(file_exists('./project/'.$this->directory.'/'.$name.'.php')){
+			$class = "project\\".$this->directory."\\".$name;
+		}
+		else{
+			$class = "base\\model\\entity\\Entity";
+		}
 		return new $class();
 	}
 	public function setDirectory($directory)
```

Comment:
I think u put a setDirectory, to allow user to change the directory (captain obvious). I call that a configurable parameters. Maybe an optional config $key?